### PR TITLE
DOC: Advertise singularity collection

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

**Comment by mih** [comment]
Canceled travis. We can just look at the README rendered in Github to confirm.

**Comment by mih** [comment]
To be merged when the container successfully built: https://singularity-hub.org/collections/667

File: README.md [hunk 1]
```diff
@@ -5,7 +5,11 @@
...
 
+<<<<<<< Updated upstream
 [![Travis tests status](https://secure.travis-ci.org/datalad/datalad.png?branch=master)](https://travis-ci.org/datalad/datalad) [![codecov.io](https://codecov.io/github/datalad/datalad/coverage.svg?branch=master)](https://codecov.io/github/datalad/datalad?branch=master) [![Documentation](https://readthedocs.org/projects/datalad/badge/?version=latest)](http://datalad.rtfd.org) [![Testimonials 4](https://img.shields.io/badge/testimonials-4-brightgreen.svg)](https://github.com/datalad/datalad/wiki/Testimonials)
+=======
+[![Travis tests status](https://secure.travis-ci.org/datalad/datalad.png?branch=master)](https://travis-ci.org/datalad/datalad) [![codecov.io](https://codecov.io/github/datalad/datalad/coverage.svg?branch=master)](https://codecov.io/github/datalad/datalad?branch=master) [![Documentation](https://readthedocs.org/projects/datalad/badge/?version=latest)](http://datalad.rtfd.org) [![https://www.singularity-hub.org/static/img/hosted-singularity--hub-%23e32929.svg](https://www.singularity-hub.org/static/img/hosted-singularity--hub-%23e32929.svg)](https://singularity-hub.org/collections/667)
+>>>>>>> Stashed changes
```
Comment (kyleam) [inline README.md]: I'm probably missing something, but just in case: Did you intend to leave the conflict markers here?

**Comment by kyleam** [comment]
> Canceled travis. 

To prevent the tests from starting at all, you could add "[ci skip]" to the commit message.

**Comment by mih** [comment]
@kyleam I thought so two, but had a case yesterday(?) where that did not work. Thanks for reminding!

File: README.md (hunk 1 above)
Comment (mih) [reply README.md]: holy cow...err.... yes, I did that obviously on purpose... SHAME SHAME SHAME ;-)

**Comment by mih** [comment]
Container built, now pulling and testing.

**Comment by mih** [comment]
@kyleam I am not sure what you are talking about ;-)

**Comment by mih** [comment]
Works!

**Comment by yarikoptic** [comment]
sweet!

**Pull request merged**